notebooks/geo/BepiColombo_FlybyVisualization.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        axs[row, col].set_title(resp_data[i]['properties']['lid'][0][33:], fontsize = 4)
</pre><p> </p><p>You can add a comment saying why you splint the lid to 33. A more explicit way could be to split the lid on : (str method split(":")) and select the piece of the lid you want to plot </p><p> </p><p>You can also add a command <code>del image_data</code> to see a the memory is freed in a better way. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>